draft-ietf-quic-transport.md

```diff
@@ -884,20 +884,18 @@ is not excessive buffering at multiple layers.
 
 ## Stream Limit Increment {#stream-limit-increment}
 
-An endpoint limits the number of concurrently active incoming streams by
-limiting the maximum stream ID.  An initial value is set in the transport
-parameters (see {{transport-parameter-definitions}}) and is subsequently
-increased by MAX_STREAM_ID frames (see {{frame-max-stream-id}}).
+An endpoint limits the number of concurrently active incoming streams.  An
+initial value is set in the transport parameters (see
+{{transport-parameter-definitions}}) and is subsequently increased by
+MAX_STREAMS frames (see {{frame-max-streams}}).  Separate limits apply to
+bidirectional and unidirectional streams.
 
 As with stream and connection flow control, this document leaves when and how
-many streams to make available to a peer via MAX_STREAM_ID to implementations,
-but offers a few considerations.  MAX_STREAM_ID frames constitute minimal
-overhead, while withholding MAX_STREAM_ID frames can prevent the peer from using
-the available parallelism.
+many streams to make available to a peer via MAX_STREAMS to implementations.
 
-The STREAM_ID_BLOCKED frame ({{frame-stream-id-blocked}}) can be
-used to signal a shortage of available streams. Implementations will likely
-want to increase the maximum stream ID as peer-initiated streams close.
```

Comment:
Are we fine with removing this sentence?

I ask this because it is the only reliable way the things would work. Adding a sentence below about an optimization (i.e. using STREAMS_BLOCKED as a signal to send to MAX_STREAMS; it is merl) at the same time removing the required interaction might confuse the readers.

Reply:
Yeah, I think that I might restore this, but attach it to the paragraph above.

Reply:
And since I see the text above, you've presumably done this.